tensorflow/stream_executor/rocm/rocm_helpers.cu.cc

```diff
@@ -31,5 +31,21 @@ void broadcast_fp32(void* stream, float* dst, int dst_stride, int batches, int s
   hipLaunchKernelGGL(broadcast_fp32_kernel, dim3(x_blocks, (batches+3)/4, src_batches), min(256, (int)size), 0,
                      (hipStream_t)stream, dst, dst_stride, batches, src, size);
 }
+
```

Comment:
I think these changes should kept out of this PR.

I updated my dedicated PR to make the existing CUDA function call our function.